Добавлена реализация в соответствии с заданием

x3qop3 opened the pull request:
ФЗ6 с новой реализацией Истории и новыми Тестами

x3qop3 committed:
Add files via upload

ФЗ6 с новой реализацией Истории и новыми Тестами

x3qop3 commented:
ФЗ6 с новой реализацией Истории и новыми Тестами

Alexander-Bolotov commented on src/practicum/InMemoryTaskManager.java:
```diff
@@ -1,197 +1,199 @@
-        return  new ArrayList<>(historyManager.getHistory());
-}
+package practicum;
+import java.util.*;
```
лучше не использовать * в импортах, а загружать только необходимые классы

Alexander-Bolotov commented on Tests/practicum/InMemoryHistoryManagerTest.java:
```diff
@@ -0,0 +1,69 @@
+        assertEquals(1, historyManager.getHistory().size(),
+                "История не должна измениться при удалении несуществующей задачи");
+    }
+}
```
добавь тест на попытку добавление null-задачи (add(null));

x3qop3 committed:
Исправил замечания

Добавил тест, нашел что использовать конкретно в import удалил лишние строки между методами
